### PR TITLE
fix: Change snippet to close logging channel

### PR DESCRIPTION
**Pull request by minherz**
Change code in Snippets.simpleLog() to call Logging.close() implicitly
Propogate exception thrown by close() to the caller wrapped in InvocationTargetException
Resolve warnings in Snippets and DeployableApplication classes.

Fixes googleapis/java-logging#676

### CODE REVIEW

**Comment by minherz** [comment]
@daniel-sanche you will have to trigger Kokoro test. It does not run automatically because it is my first submission.

File: deployable/java/src/main/java/envtest/deployable/Snippets.java [hunk 1]
```diff
@@ -63,15 +61,19 @@ public void simplelog(Map<String,String> args){
         Severity severity = getSeverity(severityString);
```
Comment (losalex) [inline deployable/java/src/main/java/envtest/deployable/Snippets.java:61]: Should the `Severity.valueOf(severityString)` used here instead? Perhaps `getSeverity()` could be rewritten to make `valueOf()` call and handle `IllegalArgumentException`.

File: deployable/java/src/main/java/envtest/deployable/Snippets.java [hunk 2]
```diff
@@ -63,15 +61,19 @@ public void simplelog(Map<String,String> args){
...
+                    .setResource(MonitoredResource.newBuilder("global").build())
+                    .build();
 
-         //Writes the log entry asynchronously
-        logging.write(Collections.singleton(entry));
+            //Writes the log entry asynchronously
+            logging.write(Collections.singleton(entry));
+        }
+        catch (Exception ex) {
+            throw new InvocationTargetException(ex, "Failed to close Logging instance");
```
Comment (losalex) [inline deployable/java/src/main/java/envtest/deployable/Snippets.java:76]: Should it be "`Failed to write log`" or something alike? I think we relate here to specific case when log cannot be closed, but `try` block might have more cases to fail?

File: deployable/java/src/main/java/envtest/deployable/Snippets.java (hunk 1 above)
Comment (daniel-sanche) [reply deployable/java/src/main/java/envtest/deployable/Snippets.java:61]: That may be worth doing if possible. I didn't realize the built in `valueOf()` already did the same thing

**Comment by daniel-sanche** [comment]
> @daniel-sanche you will have to trigger Kokoro test. It does not run automatically because it is my first submission.

Actually, there are no CI tests on this repo. You'll have to make another PR updating the Java submodule to use this commit, and the tests will run there. (I [want to look into running tests on all languages here as well](https://github.com/googleapis/env-tests-logging/issues/62), but that may be difficult)

I ran the tests manually with envctl and they looked good to me though! Thanks for finding the fix!

**Review by daniel-sanche — APPROVED** [review]

**Pull request merged**